config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -453,7 +453,7 @@ presubmits:
         - name: FEATURE_GATES
           value: '{"EventedPLEG":true}'
         - name: RUNTIME_CONFIG
-          value: '{"api/all":"true"}'
```

Comment:
I am afraid that the failure may indicate some other problems.

Reply:
/lgtm 

We may come back to see the issue as well. Another way is add another CI.
- one to enable `api/all`(alpha + beta)
- one to enable `alpha` only

Reply:
https://github.com/kubernetes/kubernetes/pull/122778#issuecomment-1928141482

Reply:
/cc @dims for approval.